design/one-pager-package-format-v2.md

```diff
@@ -219,64 +211,89 @@ spec:
-   where in the package these files exist.
-1. Any extended metadata that is opaque to Crossplane be stored in an
-   appropriately named directory starting with a period - e.g. `.upbound`, or
-   `.registry` - at the root of the package.
-
-Note that 'the package' in this context means both the content of the OCI image
-and the on-disk representation of that content; the OCI image should be built by
-simply copying the content of a directory into a layer of the OCI image. The
-base layer of packages that use this format should specify an entrypoint process
-that will, when invoked:
-
-* Emit the content of `crossplane.yaml` to stdout.
-* Discover and emit all Crossplane custom resources contained in the package to
-  stdout.
-
-The entrypoint may optionally decorate the YAML stream using any extended
-metadata (e.g. the contents of the `.registry` directory). Any such extended
-extended metadata must be written as annotations (`metadata.annotations`) on one
-or more documents in the YAML stream - it may not modify the documents in any
-other way.
-
-### Consuming an Output Stream
-
-Packages must be consumed via their output stream - a YAML stream emitted to
-stdout when the package's OCI image entrypoint is invoked. Package consumers
-must not inspect the content of the OCI image; it should be treated as opaque.
-
-All package entrypoints must support (or silently ignore) the `--context` flag.
-This flag can be used by the package consumer to provide context around how it
-is being consumed; i.e. whether the package is being consumed as part of the
-Crossplane package manager's unpack and install process or by some other system
-such as an indexer. The `--context=unpack` flag will be passed when the package
-manager unpacks a package. The `--context` flag must only affect the annotations
-(`metadata.annotations`) of zero or more documents in the output stream. It may
-not alter the stream in any other way.
+Under this format:
+
+1. An on-disk package directory contains a `crossplane.yaml` file at its root.
+1. The `crossplane.yaml` file contains either a `Provider` or a `Configuration`.
+1. The package directory contains zero or more Crossplane specific custom
+   resources (e.g. a `Composition`). Crossplane is not opinionated about where
+   under the directory these files exist.
+
+The software responsible for building an OCI image from the on-disk format is
+outside the scope of this document, but is expected to:
+
+* Append the content of `crossplane.yaml` to `package.yaml`
+* Discover and append all Crossplane resources contained to `package.yaml`
+* Create a single-layer OCI image containing `package.yaml`.
+
+The aforementioned software may also be responsible for reading additional
+metadata from the package directory and including it in the OCI image.
+
+### Additional Metadata
+
+The Crossplane package format allows for two forms of additional metadata -
+passthrough metadata and supplmentary metadata.
```

Comment:
```suggestion
passthrough metadata and supplementary metadata.
```

Reply:
It is important that our metadata be supple.